src/sandstorm/run-bundle.c++

```diff
@@ -1511,8 +1512,14 @@ private:
           backendStartTime = getTime();
         }
         if (mongoDied) {
+          // If Mongo died, let's try doing a repair. This way, every second mongo starting is a
```

Comment:
Fair enough! re: out of disk space

Then I guess there are two ways to detect this condition:

1. Rely on mongo to detect it by reading its textual log. Unfortunately,
there isn't an exit code we can rely on:
http://docs.mongodb.org/manual/reference/exit-codes/

2. Look for the file ourselves, and if it is non-empty, then always do
--repair and check that --repair exits with status code 0.

So I vote for option 2.

We could optionally do the thing where we check that the PID in that file
doesn't exist. Might as well do that, but I don't think it matters much.